modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -58,6 +58,10 @@ def set_facility_type
 
       private
 
+      def per_page
+        20
+      end
```

Comment:
why not a constant? i might be out of the loop on this repo's conventions or not aware of some best practice